### PR TITLE
Balsamic frequencies

### PR DESCRIPTION
**Pull request by dnil**
This PR adds a functionality: archive loqus frequencies for Balsamic, plus reintroduction of Swegen frequencies since Balsamic has them. Balsamic cases will be annotated with three different loqus archives:
- RD (germline) - essentially same as RD, with slightly different key names
- Cancer Germline - loqus archive for Balsamic calls for normal samples
- Cancer Somatic - loqus archive for Balsamic calls for somatic samples

<details>
<summary>Testing on cg-vm1 server (Clinical Genomics Stockholm)</summary>

**Prepare for testing**
1. Make sure the PR is pushed and available on [Docker Hub](https://hub.docker.com/repository/docker/clinicalgenomics/scout-server-stage)
1. Fist book your testing time using the Pax software available at [https://pax.scilifelab.se/](https://pax.scilifelab.se). The resource you are going to call dibs on is `scout-stage` and the server is `cg-vm1`.
1. `ssh <USER.NAME>@cg-vm1.scilifelab.se`
1. `sudo -iu hiseq.clinical`
1. `ssh localhost`
1. (optional) Find out which scout branch is currently deployed on cg-vm1: `podman ps`
1. Stop the service with current deployed branch: `systemctl --user stop scout.target`
1. Start the scout service with the branch to test: `systemctl --user start scout@<this_branch>`
1. Make sure the branch is deployed: `systemctl --user status scout.target`
1. After testing is done, repeat procedure at [https://pax.scilifelab.se/](https://pax.scilifelab.se), which will release the allocated resource (`scout-stage`) to be used for testing by other users.
</details>

<details>
<summary>Testing on hasta server (Clinical Genomics Stockholm)</summary>

**Prepare for testing**
1. `ssh <USER.NAME>@hasta.scilifelab.se`
1. Book your testing time using the Pax software. `us; paxa -u <user> -s hasta -r scout-stage`. You can also use the WSGI Pax app available at [https://pax.scilifelab.se/](https://pax.scilifelab.se).
1. (optional) Find out which scout branch is currently deployed on cg-vm1: `conda activate S_scout; pip freeze | grep scout-browser`
1. Deploy the branch to test: `bash /home/proj/production/servers/resources/hasta.scilifelab.se/update-tool-stage.sh -e S_scout -t scout -b <this_branch>`
1. Make sure the branch is deployed: `us; scout --version`
1. After testing is done, repeat the `paxa` procedure, which will release the allocated resource (`scout-stage`) to be used for testing by other users.
</details>

**How to test**:
1. how to test it, possibly with real cases/data

**Expected outcome**:
The functionality should be working
Take a screenshot and attach or copy/paste the output.

**Review:**
- [ ] code approved by
- [x] tests executed by

### CODE REVIEW

File: scout/build/variant/variant.py [hunk 1]
```diff
@@ -249,6 +249,22 @@ def build_variant(
     variant_obj["local_obs_old_desc"] = variant.get("local_obs_old_desc")
     variant_obj["local_obs_old_nr_cases"] = variant.get("local_obs_old_nr_cases")
 
+    # local observations from cancer pipeline
+    variant_obj["local_obs_cancer_germline_old"] = variant.get("local_obs_cancer_germline_old")
```
Comment (northwestwitch) [inline scout/build/variant/variant.py:253]: Here it is a bit confusing for me. Like what's the difference between `local_obs_cancer_germline_old` and `local_obs_cancer_germline_old_freq`etc? Could it be clarified either by comment or better by renaming these strings to something more user friendly? Is `local_obs_cancer_germline_old` the number of observarions? Then perhaps rename it to `local_obs_cancer_germline_old_obs`?

File: scout/server/blueprints/variant/templates/variant/variant_details.html [hunk 2]
```diff
@@ -202,17 +202,33 @@
...
           </tr>
         </thead>
         <tbody>
           <tr>
+            <td>RD</td>
             <td>{{ variant.local_obs_old|default('N/A') }}</td>
             <td>{{ variant.local_obs_hom_old|default('N/A') }}</td>
             <td>{% if variant.local_obs_old_freq %} {{variant.local_obs_old_freq|round(6)}} {% elif variant.local_obs_old_nr_cases and variant.local_obs_old %} {{ (variant.local_obs_old/variant.local_obs_old_nr_cases)|round(5) }} {% elif variant.local_obs_old_nr_cases %} 0 / {{variant.local_obs_old_nr_cases}} {% else %} N/A {% endif %}</td>
           </tr>
+          {% if variant.category == "cancer" %}
```
Comment (northwestwitch) [inline scout/server/blueprints/variant/templates/variant/variant_details.html:218]: This is very nice! 💯

**Review by northwestwitch — APPROVED** [review]
It's working so I'm approving. Would be nice to try to make it a bit easier for the reviewer and for future maintenance to rename vars to be more explicit, see my comments!

File: scout/build/variant/variant.py (hunk 1 above)
Comment (dnil) [reply scout/build/variant/variant.py:253]: Agree in principle, but I did want them consistent with the RD ones, and didn't want to change those keys. I'll add some more comments!

**Comment by dnil** [comment]
This is still missing the filtering option as per #4017, which is only partially solved. I so slowly feel done for today - let's make it a two step thing.

**Pull request merged**